index.html

```diff
@@ -165,7 +165,8 @@ <h2 class="bannerbox__title">I need a ride</h2>
                             <div class="form-column">
                                 <div class="form-group">
                                     <label>Find your polling place</label>
-                                    <ul id="location-details" class="state-dropdown">
+									<div class="help-block">(Link will open your state’s polling place checker in a new tab.)</div>
```

Comment:
Nice call using the `help-block` class, I hadn't thought of that. Looks like it works perfectly.

The indentation on this isn't quite right - it appears to be using tabs instead of spaces, which isn't a huge deal for me personally, bit it's affecting the indentation so that it looks like this to me:
![image](https://cloud.githubusercontent.com/assets/1155816/19694673/3229527e-9ad8-11e6-9456-67b5375a3e95.png)

Also I would've used a <p> tag instead of a div, but I don't think it really matters.